sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSenderOptions.cs

```diff
@@ -9,17 +9,8 @@ namespace Azure.Messaging.ServiceBus
     /// The set of options that can be specified when creating a <see cref="ServiceBusSender"/>
     /// to configure its behavior.
     /// </summary>
-    public class ServiceBusSenderOptions
+    internal class ServiceBusSenderOptions
```

Comment:
I'm assuming we are just keeping this type for potential future use?

Reply:
Yes